### ISSUE TITLE
Same Input node hits assert error when attempting to re-quantize bias.

### ISSUE THREAD

**Issue by vixadd** [neuralmagic/sparseml#390]
**Describe the bug**
Nodes in a hierarchical structure from a keras model to ONNX model, hits an assert error when trying to quantize input bias.

**Expected behavior**
ONNX quantizer doesn't re-use layers to quantize models in the structure. Layers are quantized one at a time.

**Environment**
Include all relevant environment information:
1. OS [e.g. Ubuntu 18.04]: Ubuntu 20.04
2. Python version [e.g. 3.7]: 3.6
3. SparseML version or commit hash [e.g. 0.1.0, `f7245c8`]: 0.7.0 -- https://github.com/vixadd/sparseml/commit/c5b4e37c404ace0c8d4ee046480a3705f7551235
4. ML framework version(s) [e.g. torch 1.7.1]: Tensorflow 2.3.0 -- Keras Model
5. Other Python package versions [e.g. SparseZoo, DeepSparse, numpy, ONNX]: ONNX 1.10.1
6. Other relevant environment information [e.g. hardware, CUDA version]:

**To Reproduce**
-- Run a keras model that reuses multiple heirarchical convolution layers. Convert to ONNX and run that through the quantizer function in the onnx.optim library.

**Errors**
Assert Error: -- (In my code posted above.)
```bash
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_3_1/init/conv2d_3_1/Conv2D_bias_fused_bn already has a quantization value.
WARNING: wrapped_model/call/encoder_1/init/conv2d_init/init/conv2d_init/Conv2D_bias_fused_bn already has a quantization value.
```

**Additional context**
I have already tried renaming my layers to unique identifiers. It doesn't work. The layers, while unique, get reused when quantizing bias. I am confused as to why the quantizer re-uses conv layers, and then complains about the fact that layers are being re-used. Please tell me if I'm missing something. -- Thanks.

**Comment by markurtz** [comment]
Hi @vixadd, thank you for raising this issue. Could you point us towards a specific open-sourced model or implementation we could look at further? Additionally, what was the method you used for conversion to ONNX? We can then go through and verify the issue from our side to work on a specific fix for this issue. 

Thanks,
Mark

**Comment by vixadd** [comment]
Since this has gone a little stale, I'll put up an example, and just leave it at that.
```python
import tensorflow.keras as tfk

def build_encoder():
    layers = []
    for l in rang(0, 10):

        layer = []
        for b in range(0, 5):
            layer.append( tfk.layers.Conv2D(16, 8, 64, use_bias=False, name=f"encoder/conv2d_{l}_{b}" )

        layers += [layer]
```
Building something like that into your model will cause this error.

**Comment by markurtz** [comment]
Hi @vixadd, thank you for putting up that pull request! The code has been merged in now and will be going out with our 0.8 release. It will be available on nightly starting tonight, so I'm going to close this out for now. If any issues come up feel free to reopen.

Thanks,
Mark
